macros/elvish_macros.cfg

```diff
@@ -217,6 +217,49 @@
         (_"All right, we will wait for the spring.") (no)}
         {IF ano_opt equals yes}
         {MESSAGE (Kyobaine) (portraits/kyobaine.png) (Kyobaine) _"You think you can bribe me, an elvish druid, with gold? Oh well. This will allow me to buy better winter clothes."}
+        {MSGOPTI2 (Gawen Hagarthen) (portraits/gawen_winter.png) (_"Maybe I should act gentlemanly here?")
```

Comment:
Well this passage reads a little OOC but since it seems more like for fun let's just leave it.

Reply:
Ok, cool.